City-of-Helsinki/hauki-admin-ui
# test: add playwright tests

## What?
This commit adds additional Playwright tests

## How to test

Run the tests locally with `yarn test:e2e:start` and verify that all tests pass ✅
